client/components/jetpack-logo/index.jsx

```diff
@@ -3,11 +3,47 @@
  */
 import React from 'react';
 
-const JetpackLogo = ( { size } ) => {
-	const svgPath = 'M42,0A42,42,0,1,0,84,42,42,42,0,0,0,42,0ZM40.36,' +
+const JetpackLogo = ( { size, full } ) => {
+	const svgIconPath = 'M42,0A42,42,0,1,0,84,42,42,42,0,0,0,42,0ZM40.36,' +
```

Comment:
At this point it's pretty strange to keep this path separate from it's SVG. It's pretty far from where it's used and it's just static. Could we inline it below?

Reply:
Yes. I don't know why it was like that in the first place.